src/main/groovy/org/shipkit/gradle/UpdateReleaseNotesTask.java

```diff
@@ -321,6 +335,11 @@ void setIncrementalNotesGenerator(IncrementalNotesGenerator incrementalNotesGene
         this.incrementalNotesGenerator = incrementalNotesGenerator;
     }
 
+    @ExposedForTesting
+    void setHeaderRemover(HeaderRemover headerRemover){
```

Comment:
If you rebase, you will find this simpler and we may not need "ExposedForTesting" any more. I moved the internal implementation out of the task. In this merge race, my code went first, please rebase :)